trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/CST/airfoil_from_CST.py

```diff
@@ -0,0 +1,137 @@
+## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil-CST
```

Comment:
this is not a method. This is a class which should be a component attribute. See this folder:

https://github.com/suavecode/SUAVE/tree/develop/trunk/SUAVE/Components/Wings/Airfoils